### DISCUSSION TITLE
activating two buttons at a time

### DISCUSSION THREAD

**Discussion by Starfiretbt** [AntiMicroX/antimicrox#806] [Q&A]
I'm playing a flight simulator and I want to assign one button to do the task of shift+F10 however it only seems to do F10. I had under advanced selected [Shift] and [F10] but that didn't work. I then saw the "join" option so I set the box to [Shift+F10], still only does the action of F10. F10 and Shift+F10 do two different things in this game.

I have not done a search for similar discussions because I don't know what to search for, pressing multiple buttons?

**Comment by pktiuk** [comment]
This seems to be bug, firstly try with other keyboard buttons like A+B

**Reply by Starfiretbt** [reply]
It works for other keys, for example assigning the letter [a] gets me "a" when pressed while [shift+a] gets me "A" uppercase. However [shift F10] still gets me the action of F10.
The game is "Star Wars: X-wing" and F10 sets shield recharge rate while Shift+F10 causes lasers to be dumped into shields, they are two different ways of recharging shields. The keyboard commands work just not when assigned to the joystick through Antimicro X

**Reply by pktiuk** [reply]
In this case you should create a proper bug report in the issues page of this repository. https://github.com/AntiMicroX/antimicrox/issues/new/choose
